src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPart.DetailAdmin.cshtml

```diff
@@ -12,20 +12,31 @@
 }
 else
 {
-    <p>
-        <div class="dropdown">
-            <a class="btn btn-primary dropdown-toggle" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                @T["Create"]
+    if (Model.ContainedContentTypeDefinitions.Count() == 1)
+    {
```

Comment:
Create a variable `contentTypeDefinition` and reuse it instead of calling `Model.ContainedContentTypeDefinitions.FirstOrDefault()` multiple times.